README.md

```diff
@@ -6,14 +6,14 @@
 
 - [Quick Start](./docs/quick-start.md)
 
-Load Balancer Operator for Kubernetes is a Cluster API speaking operator for load balancers.
+Load Balancer Operator for Kubernetes is a Cluster API speaking operator for load balancers. It manages the lifecycle of the [Load Balancer and Ingress Service](https://github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/tree/master) resources for Kubernetes clusters.
```

Comment:
We may want to support other load balancer implementations, we can modify this statement to:
> It manages the lifecycle of load balancers implementations and provides a cluster control plane high availability interface in the multi-cluster scenario.

Reply:
changed